src/actions/Bluetooth/UpdateActions.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
using this throughout the app as `UpdateActions.set..` doesn't sit right, sorry!

Maybe.. `SensorUpdate` or something..? Am I being ridic?

Reply:
Ah, I think I see why it might be a bit weird - there does seem to be a bit of a lack of context when looking at the code without the file structure available? 🤔 . Perhaps we could prefix these all with `Sensor`? Even `Bluetooth/Update` is a little vague thinking about it, could be dealing with Bluetooth headphones or keyboards...or something else.

Reply:
Sorry didn't think about this before. I suppose generally I don't look at the file structure much, I think that's why I like to export objects like the `XXActions` objects as it gives context to whatever you're importing.

There's certainly a possibility of using bluetooth keyboards and bluetooth barcode scanners in the future, so maybe you're right.